### PR TITLE
import-vm: remove username, password request params for vm import

### PR DESCRIPTION
**Pull request by shwstppr**
Based on https://github.com/apache/cloudstack/issues/8414

----
📚 Documentation preview 📚: https://cloudstack-documentation--401.org.readthedocs.build/en/401/

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]
lgtm (when https://github.com/apache/cloudstack/pull/9124 is merged

File: source/adminguide/virtual_machines/importing_unmanaging_vms.rst [hunk 1]
```diff
@@ -481,8 +479,6 @@ choose the temporary storage location on the external host for the converted fil
 .. parsed-literal::
    - **zoneid** (Zone to which Instance will be imported)
    - **host** (the host name or IP address of External Host)
-   - **username** (the username of External Host for authentication)
-   - **password** (the password of External Host for authentication)
```
Comment (sureshanaparti) [inline source/adminguide/virtual_machines/importing_unmanaging_vms.rst:485]: @shwstppr these are required here

File: source/adminguide/virtual_machines/importing_unmanaging_vms.rst [hunk 2]
```diff
@@ -437,8 +437,6 @@ those same credentials are later used for SSH authentication when the QCOW2 imag
 .. parsed-literal::
    - **zoneid** (Zone to which Instance will be imported)
    - **host** (the host name or IP address of External Host)
```
[the remark this answers is not in the document]
Comment (shwstppr) [reply source/adminguide/virtual_machines/importing_unmanaging_vms.rst:439]: I don't think adding them even as optional right now gives any benefit as they are completely unused for listing

[the remark this answers is not in the document]
Comment (sureshanaparti) [reply source/adminguide/virtual_machines/importing_unmanaging_vms.rst:439]: correct @shwstppr, remove them - not required for listing

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**